addons/web/static/src/views/fields/monetary/monetary_field.js

```diff
@@ -72,16 +79,20 @@ export class MonetaryField extends Component {
         return this.props.record.data[this.props.name];
```

Comment:
is this.value still necessary? Shouldn't we always use this.state.value (which would ensure that the input and the span are always synchronized)